pcap-linux.c

```diff
@@ -2426,8 +2426,8 @@ pcap_stats_linux(pcap_t *handle, struct pcap_stat *stats)
 	 *	"ps_drop" is not supported.
 	 *
 	 *	"ps_ifdrop" is supported. It will return the number
-	 *	of drops the interface reports in /proc/net/dev,
-	 *	if that is available.
+	 *	of drops the interface reports in /sys/class/net, if that
```

Comment:
Perhaps that should be "...the interface reports in /sys/class/net/statistics/{if_name}/rx_{missed,fifo}_errors", as per another comment.